promgen/templates/promgen/service_action_button_group.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+{% load promgen %}
+
+
+<div style="display: inline-block;">
+  <div class="btn-group btn-group-sm" role="group" aria-label="...">
+    <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      {% translate "Actions" %} <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu">
+      <li role="presentation"><a href="{% urlqs 'audit-list' service=service.id %}">{% translate "Edit History" %}</a></li>
+      <li role="presentation"><a href="{% urlqs 'alert-list' service=service.name %}">{% translate "Alert History" %}</a></li>
+
+      <hr>
+
+      <li role="presentation"><a href="{% url 'api:service-rules' name=service.name %}">{% translate "Export Rules" %}</a></li>
+      <li role="presentation"><a href="{% url 'api:service-targets' name=service.name %}">{% translate "Export Service" %}</a></li>
+
+      <hr>
+
+      <li role="presentation">
+        <a href="">
+          <form method="post" action="{% url 'service-delete' service.id %}" onsubmit="return confirm('{% translate "Delete this service?" %}')" style="display: inline">
+            {% csrf_token %}   
+              <button style="background:none;border: none;padding: 0;">{% translate "Delete Service" %}</button>  
+          </form>
+        </a>
+      </li>
+
+      <hr>
+
+      <li>
+        <a href="">
+          <form action="{% url 'service-notifier' service.id %}" style="display:inline" method="post" v-pre>{% csrf_token %}
+            <input type="hidden" name="sender" value="promgen.notification.user">
+            <input type="hidden" name="value" value="{{request.user.username}}" />
+            <button style="background:none;border: none;padding: 0;">{% translate "Subscribe to Notifications" %}</button>
+          </form>
+        </a>
+      </li>
+
+    </ul>
```

Comment:
<img width="259" alt="スクリーンショット 2024-07-29 12 07 50" src="https://github.com/user-attachments/assets/b250a239-21ef-4119-aa26-09e0ff1c7c74">

We may want to consider the order.

`Subscribe to Notifications` feels like it could be moved up to the top.
`Delete Service` is a dangerous action. And though it does have a confirmation popup, it feels like we should put it at the very bottom, but also change the color to show it's a dangerous action. (We should probably have it go to a specific `delete confirmation` page instead of *just* the confirmation popup but that's outside the scope of this PR)

Reply:
Ok. I update the order and change the color for `Delete Service`